### PR TITLE
Issue #13809: Kill mutation in Tree-Walker

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13809: Kill mutation in Tree-Walker

Test added

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java [hunk 1]
```diff
@@ -536,6 +537,33 @@ public void testMultiCheckOrder() throws Exception {
                 expected);
     }
 
+    @Test
+    public void testMultiCheckOrder2() throws Exception {
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java:541]: testMultiCheckOfSameTypeNoIdResultsInOrderingByHash

**Review by romani — CHANGES_REQUESTED** [review]
Goal of test is not clear if pitest diff not visible.

Item:

File: src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**